examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Program.cs

```diff
@@ -39,12 +49,17 @@ public static void Main(string[] args)
                             builder.AddAzureAppConfiguration(options =>
                             {
                                 options.Connect(new Uri(appConfigurationEndpoint), new DefaultAzureCredential())
+                                        .ConfigureKeyVault(kv =>
+                                        {
+                                            kv.SetCredential(new DefaultAzureCredential());
+                                        })
```

Comment:
Please move this after the `.ConfigureRefresh` call.

Reply:
Moved and renamed.

---
In reply to: [427591529](https://github.com/Azure/AppConfiguration/pull/329#discussion_r427591529) [](ancestors = 427591529)